.github/workflows/scrape.yml

```diff
@@ -3,26 +3,23 @@ on:
   repository_dispatch:
     types: scrape
   # TODO Ideally this will only run when changes to the scraper folder are made
-  # push:
-  #   branches:
-  #     - master
+  push:
+    branches:
+      - master
   schedule:
     - cron: '0 * * * *'
   workflow_dispatch:
+    inputs:
+      scrape_all:
+        description: 'Scrape all semesters (ALL_YEARS|CURRENT_YEAR)'
+        required: true
+        default: 'CURRENT_YEAR'
```

Comment:
Is it possible to set this to be `ALL_YEARS` if the scrape is run off of a push rather than the cron job?  And if so, do we want to do that?

Reply:
I dont know how to do that, but in theory we can. Although I think this should be split out into its own PR along with fixing the TODO at the top of the file to run this script on changes to the scraper folder but not the entire repo.